include/robot_interfaces/robot_log_reader.hpp

```diff
@@ -0,0 +1,65 @@
+/**
+ * @file
+ * @brief API to read the data from a robot log file.
+ * @copyright 2020, Max Planck Gesellschaft. All rights reserved.
+ * @license BSD 3-clause
+ */
+#pragma once
+
+#include <fstream>
+#include <vector>
+
+#include <cereal/archives/binary.hpp>
+#include <cereal/types/tuple.hpp>
+#include <cereal/types/vector.hpp>
+
+#include <robot_interfaces/robot_log_entry.hpp>
+
+namespace robot_interfaces
+{
+/**
+ * @brief Read the data from a robot log file.
+ *
+ * The data is read from the specified file and stored to the `data` member
+ * where it can be accessed.
+ */
+template <typename Action, typename Observation>
+class RobotLogReader
```

Comment:
I think we should specify this reader is for binary file type o/w it may be a bit confusing

Reply:
done